.github/workflows/docs-lint.yml

```diff
@@ -0,0 +1,26 @@
+name: docs-lint
+
+on:
+  push:
+    branches: [ "*" ]
+    paths:
+      - '.github/workflows/docs-lint.yml'
+      - 'docs/content/**'
+  pull_request:
+    branches: [ "*" ]
+    paths:
+      - '.github/workflows/docs-lint.yml'
+      - 'docs/content/**'
+
+jobs:
+  spelling:
+    name: Spell Check
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
```

Comment:
Depending on the size of the spellcheck-cli bundle, we might want to enable caching, if it's not done by default. In this case, I'm not sure whether npx targets the right directory if the package is not there. However, if the package is already installed, I think it uses it instead of fetching it again. So we might need to keep the npm install in the GitHub Action. To be tested

Reply:
I have tested by uninstalling `spellcheck-cli` and running npx. I looked like npx grabbed it into a temp spot and ran it. Did not appear in my global modules list after, but ran faster on future executions. Perhaps a different cache location than the standard global node modules cache.
I can perform these enhancements once GHA runs and we observe the behaviour :
- Re-enable npm install
- Activate GHA node-modules cache with `cache: 'npm'`